### DISCUSSION TITLE
gettext ext

### DISCUSSION THREAD

**Discussion by mshannaq** [beyondcode/herd-community#279] [General]
does Herd come without php gettext? becuae I got error `Call to undefined function bindtextdomain()` which is related to gettext

`php -i | grep gettext` show

notthing

I installed Herd on Mac M2

**Comment by hnccox-yard** [comment]
I am having the same issue!

What I did now was install php@7.4 with homebrew and symlink to this....

@mshannaq Did you find a solution??

**Reply by mpociot** [reply]
Unfortunately gettext is not (yet) supported. Maybe @crazywhalecc knows if this actually can be implemented into the static PHP build

**Comment by mshannaq** [comment]
> I am having the same issue!
> 
> What I did now was install php@7.4 with homebrew and symlink to this....
> 
> @mshannaq Did you find a solution??

@hnccox-yard really no i did not find a solution , i return back to valet until I can investigate herd in the future.

**Reply by hnccox-yard** [reply]
@mshannaq 
I am currently using Herd, but instead of symlinking to php74, I made php74 symlink to homebrew php@7.4

So I can still switch using Herd, but under the hood it's actually using the homebrew installed php versions
